### DISCUSSION TITLE
Windows Setup

### DISCUSSION THREAD

**Discussion by jcrookwa** [tommyblue/smugmug-backup#45] [Q&A]
Apologies for the basic question but what is the process for setting this up on windows?
Any help would be appreciated! 

Thanks

**Comment by tommyblue** [comment]
Hi @jcrookwa, it's not different from the other operating systems and it should be well detailed in the README (shown in the main page of the project). In short:

1. download the latest release from the [release page](https://github.com/tommyblue/smugmug-backup/releases) (pick the correct version for your o.s.)
2. in the same folder of the downloaded binary, create the `config.toml` file (you can copy the `config.example.yml` file) and customize it. On windows, note that backslashes (`\`) must be doubled (`\\`). This generally happens in paths.
3. run the program. This can be done double-clicking on the executable or, suggested for the first run, from the command line. Doing so you can check the logs and quickly spot configuration errors
